web/src/rest.js

```diff
@@ -0,0 +1,14 @@
+
+import { RestClient } from '@girder/components/src';
+
+const apiRoot = process.env.VUE_APP_API_ROOT || 'http://localhost:8080/api/v1';
+const girderRest = new RestClient({ apiRoot, setLocalCookie: true });
+
+const loggedIn = () => !!girderRest.user;
+const user = () => girderRest.user;
+
+export {
```

Comment:
@AlmightyYakob Is there a reason you prefer this style of `export`, rather than just defining the exports inline with the declarations (e.g. `export const user = ...`)?